src/app/news/news.component.ts

```diff
@@ -1,15 +1,29 @@
 import { Component, OnInit } from '@angular/core';
+import { NewsItem } from '../core/model/news/NewsItem';
+import { GatewayService } from '../core/service/gateway.service';
+import { LoaderService } from '../shared/spinner/loader.service';
 
 @Component({
   selector: 'app-news',
   templateUrl: './news.component.html',
   styleUrls: ['./news.component.scss']
 })
 export class NewsComponent implements OnInit {
+  limit = '100';
```

Comment:
why 100? add comment please
```suggestion
  limit = '100'; // please comment me
```

Reply:
ah, I see, it's about pagination, but something went wrong with that.

Reply:
See this section in documentation https://www.npmjs.com/package/ngx-pagination#server-side-paging